source/help/messaging/attaching-files.rst

```diff
@@ -3,62 +3,77 @@ Attaching Files
+After dropping the files, press ENTER to post.
+
+Paste from the Clipboard
 ~~~~~~~~~~~~~~
-On Chrome and Edge browsers, it is also possible to upload files by
-pasting them from the clipboard. This is not yet supported on other
-browsers.
+If you use Chrome or Edge browsers, you can also upload files by 
+pasting them from the clipboard. This feature is not yet supported on 
+other browsers.
 
 File Previewer
 --------------
-Mattermost has a built in file previewer that is used to view media,
-download files and share public links. Click the thumbnail of an
-attached file to open it in the file previewer.
+Mattermost has a built-in file previewer that you can use to:
 
-Sharing Public Links
-~~~~~~~~~~~~~~~~~~~~
-Public links allow you to share file attachments with people outside
-your Mattermost team. Open the file previewer by clicking on the
-thumbnail of an attachment, then click **Get Public Link**. This opens a
-dialog box with a link to copy. When the link is shared and opened by
-another user, the file will automatically download.
+-  Download files
+-  Share public links
+-  View media
 
-If **Get Public Link** is not visible in the file previewer and you
-prefer the feature enabled, you can request that your System Admin
-enable the feature from the System Console under **Security** > **Public
-Links**.
+Click the thumbnail of an attached file to open it in the file previewer.
 
 Downloading Files
 ~~~~~~~~~~~~~~~~~
-Download an attached file by clicking the download icon next to the file
-thumbnail or by opening the file previewer and clicking **Download**.
+You can download an attached file in two ways:
+
+-  click the download icon next to the file thumbnail
+-  open the file previewer and click **Download**
+
+[screenshots of download methods]
+
+Sharing Public Links
+~~~~~~~~~~~~~~~~~~~~
+Public links allow you to share attachments with people outside your 
+Mattermost team. Open the file previewer by clicking on the thumbnail of
+an attachment, then click **Get Public Link**. This opens a dialog box with 
+a link to copy. When the link is shared and opened by another user, the 
+file will automatically download.
+
+[Change to a Tip:] If **Get Public Link** is not visible in the file previewer
+and you prefer that the feature is enabled, you can request your System 
+Administrator to enable the feature from the System Console under 
+**Security** > **Public Links**.
 
-Supported Media Types
+Viewing Media:
 ~~~~~~~~~~~~~~~~~~~~~
-If you are trying to preview a media type that is not supported, the
-file previewer will open a standard media attachment icon. Supported
-media formats depend heavily on your browser and operating system, but
-the following formats are supported by Mattermost on most browsers:
+If you are trying to preview a media type that is not supported, the 
+file previewer will open a standard media attachment icon. Supported 
+media formats depend on your browser and operating system, but the 
```

Comment:
I'm not sure I understand what is meant by "open a standard media attachment icon"